server/auth/types/openid/routes.ts

```diff
@@ -163,7 +184,10 @@ export class OpenIdAuthRoutes {
 
         // authHeaderValue is the bearer header, e.g. "Bearer <auth_token>"
         const token = cookie?.credentials.authHeaderValue.split(' ')[1]; // get auth token
-        const requestQueryParameters = `?post_logout_redirect_uri=${this.getBaseRedirectUrl()}/app/kibana`;
+        const requestQueryParameters = `?post_logout_redirect_uri=${getBaseRedirectUrl(
+          this.config,
+          this.core
+        )}/app/kibana`;
```

Comment:
Do we need /app/kibana here? I suppose server will server auto redirect to it?

Reply:
you are right, it is not necessary, removed